recipe/meta.yaml

```diff
@@ -11,18 +11,17 @@ source:
   sha256: d435f21b6f74ba42669e390d43a3b6ab401ec3a9420e7fb13d7971a7a71c3c0e
 
 build:
-  number: 1
+  number: 2
   skip: true   # [win]
 
 requirements:
   build:
-    - boost 1.60.*
-    - netcdf-cxx4
     - cmake
+    - boost 1.61.*
+    - netcdf-cxx4 4.3.*
   run:
-    - boost 1.60.*
-    - hdf5 1.8.15*
```

Comment:
If we pin `netcdf-cxx4` we don't need to add the `hdf5` pin here b/c the right `libnetcdf`/`hdf5` will be downloaded.